index.html

```diff
@@ -555,30 +555,65 @@ <h2>
         mode</a>.
       </p>
       <p>
-        Each <a>display mode</a>, except <code>browser</code>, has a
-        <dfn>fallback display mode</dfn>, which is the <a>display mode</a> that
-        the user agent can try to use if it doesn't support a particular
-        <a>display mode</a>. If the user agent does support a <a>fallback
-        display mode</a>, then it checks to see if it can use that <a>display
-        mode</a>'s <a>fallback display mode</a>. This creates a fallback chain,
-        with the <a>default display mode</a> (<code>browser</code>) being the
-        last item in the chain.
+        The <dfn>fallback chain</dfn> for:
```

Comment:
I think we should have a sentence defining fallback chain first. This allows other extensions to add their own (rather than the definition of FC literally being this set of 4 literal ones).

"Every `{{`display mode`}}` has a `<dfn>`fallback chain`</dfn>`, which is a list of `{{`display modes`}}`. The `{{`fallback chain`}}` for:"